README.md

```diff
@@ -25,13 +25,13 @@ Or install it yourself as:
 
 You can add a check to a webservice when configuring heartcheck
 
-The service is a Hash that needs to respond to `:name` to identify the service, `:url` of the service (GET request) and `:body_match` is a regex that is going to match the response body.
+The service is a Hash that needs to respond to `:name` to identify the service and `:url` of the service (GET request).
```

Comment:
what do you think to say the `body match` is optional?

Reply:
I thought about writing it there. But since there's an example using all the available options (including `body_match`), I thought it wasn't necessary. Do you think there should be another section in the README listing all other options?

Reply:
maybe you can list the other options and link to a place for more details (maybe the source code)

wdyt?